backend/schema.sql

```diff
@@ -191,3 +191,19 @@ CREATE TABLE redeployment_events (
 	message TEXT,
 	created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
+
+CREATE TABLE notice (
```

Comment:
テーブル名は複数形にしているので合わせてほしいです
```suggestion
CREATE TABLE notices (
```